src/naive/datetime/mod.rs

```diff
@@ -126,12 +126,11 @@ impl NaiveDateTime {
         let datetime = NaiveDateTime::from_timestamp_opt(secs, nsecs);
         datetime.expect("invalid or out-of-range datetime")
     }
-
-    /// Makes a new `NaiveDateTime` corresponding to a UTC date and time,
-    /// from the number of milliseconds
-    /// since the midnight UTC on January 1, 1970 (aka "UNIX timestamp").
+    /// Creates a new [NaiveDateTime] from milliseconds since UNIX Epoch.
```

Comment:
Both here and in the next line, I think prefixing "UNIX" with "the" ("the UNIX epoch") would read better.

Reply:
Sure, done,